Olaheavy2021/registrationApp
# Design and Style the Course Detail Page and load the list of modules

# Ticket Number and description
[CBP-30: Properly Styled Pages and Load Dynamic Data for Course List and Course Detail](https://sunkanmiolawuwogas.atlassian.net/browse/CBP-30)

## Context
As a user, I want the course list and course detail pages to be styled appropriately and visually appealing. This will enhance the user experience and make the navigation and consumption of course-related information more enjoyable.
The pages should also be populated with dynamic content from the database.

## Work Done

- Design and Style the Course List Page and load courses from the database
- Design and Style the Course Detail Page and load the list of modules from the database
- Apply Consistent Styles to all Pages

## Screenshots
<img width="853" alt="course_details" src="https://github.com/Olaheavy2021/registrationApp/assets/86934237/faca0ea4-f005-4feb-952e-d41d4b91ccda">
<img width="950" alt="course_list" src="https://github.com/Olaheavy2021/registrationApp/assets/86934237/3d7fd25c-9349-48d0-8b62-de68044f1e04">
<img width="883" alt="course-details-404" src="https://github.com/Olaheavy2021/registrationApp/assets/86934237/92d7b573-04e9-4bfd-9d62-d7db7a21f545">


## Testing instructions
- Navigate to the Courses pages
- The courses should be listed on it 
- Click on one of the courses, the user should be redirected to the course_details page.

## Acceptance Criteria

- [x] The course list page is styled correctly, with organized and visually appealing course cards or listings.
- [x] The course detail page has an attractive layout, presenting course information in a straightforward and visually pleasing manner.
- [x] The styling is consistent across the course list, course detail, and course modules pages, maintaining a cohesive design.
